commit ce5ded42fc6d9d740d512d24c9f05894dbddac7f
Author: Milton Eiji Takamura <miltontakamura@gmail.com>
Date:   Wed Mar 9 09:38:36 2022 +0000

    Customizations

diff --git a/config.def.h b/config.def.h
index e06be66..1d33fa8 100644
--- a/config.def.h
+++ b/config.def.h
@@ -62,6 +62,15 @@ static const char unknown_str[] = "n/a";
  * wifi_essid          WiFi ESSID                      interface name (wlan0)
  */
 static const struct arg args[] = {
-	/* function format          argument */
-	{ datetime, "%s",           "%F %T" },
+	/* function      format            argument */
+  { ram_used,      "[ RAM %s ] ",    NULL },
+  { cpu_perc,      "[ CPU %s%%",       NULL },
+  { cpu_freq,      " %s ] ",         NULL },
+  { wifi_essid,    "[ %s",           "wlp5s0" },
+  { wifi_perc,     " %s%% ] ",         "wlp5s0" },
+  { run_command,   "[ %s ] ",        "public-ip" },
+  //{ run_command,   "[ vol: %4s ] ",  "amixer sget Master | awk -F\"[][]\" '/%/ { print $2 }' | head -n1" },
+  //{ battery_perc,  "[ bat: %s%% ",   "BAT0" },
+  //{ battery_state, "%s ] ",          "BAT0" },
+	{ datetime,      "[ %s ] ",         "%a, %b %d %H:%M" },
 };
